internal/testutils/testutils.go

```diff
@@ -45,3 +75,15 @@ func DefaultSucceedingCandidates() []types.MetaBundle {
 
 	return res
 }
+
+func NewBundle(name string, yamlFilePaths ...string) registry.Bundle {
```

Comment:
Nice !

One more thing, are we going to load bundles from assets located in a different directory? Thinking we could add `path.Join(BundleAssetsDir, "amXXXX/<filename>.yaml")` inside `NewBundle`

and maybe provide  a comment

```go
// NewBundle - create bundle from assets in `internal/testdata/assets/`
```

Reply:
Good stuff! Let me fix this :)

Reply:
> path.Join(BundleAssetsDir, path) inside NewBundle, where path is relative (e.g.: "am0007/csv.yaml")

Wouldn't this a bit confusing ( kinda over-abstracted) for the validator developer?
Because, I guess that from the eyes and perspective of the validator developer, while coding out the `test_am0007.go` they would be working inside the `pkg/validators/` directory in that specific moment. And from their eyes, the assets files would sit inside "../../internal/testdata/assets/am0007/" and that's the first thing which will pop in their mind to feed as the yamlFilePaths.
But just "am0007/*" would feel weird unless they exactly know that behind the scenes, the `NewBundle` function is prefixing their input path with "../../internal/testdata/assets". 

Sounds a lot for the validator developer (considering myself to be in the shoes of one) but wdyt :)

Reply:
that sounds perfectly reasonable. Lets merge it ;)